control-plane/api-gateway/consul/config_entry_translation.go

```diff
@@ -0,0 +1,91 @@
+package consul
+
+import (
+	"os"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/types"
+	gwv1beta1 "sigs.k8s.io/gateway-api/apis/v1beta1"
+
+	"github.com/hashicorp/consul-k8s/control-plane/namespaces"
+	"github.com/hashicorp/consul/api"
+	consulAPI "github.com/hashicorp/consul/api"
```

Comment:
Generally, we will call this `capi` in the Consul on Kubernetes codebase.

Reply:
updated!